Modules/File/Sources/File/File.swift

```diff
@@ -175,7 +157,7 @@ extension File {
 
     private func ensureFileIsOpen() throws {
         if closed {
-            throw StreamError.closedStream(data: Data())
+            throw StreamError.closedStream(buffer: Buffer())
```

Comment:
this could be `Buffer.empty` right? Maybe find + replace `Buffer()` -> `Buffer.empty`.